main/src/ca/uwaterloo/flix/language/ast/Symbol.scala

```diff
@@ -122,15 +122,15 @@ object Symbol {
   /**
     * Returns a fresh type variable symbol with the given text.
     */
-  def freshKindedTypeVarSym(text: VarText, kind: Kind, isRegion: Boolean, loc: SourceLocation)(implicit scope: Scope, flix: Flix): KindedTypeVarSym = {
-    new KindedTypeVarSym(flix.genSym.freshId(), text, kind, isRegion, scope, loc)
+  def freshKindedTypeVarSym(text: VarText, kind: Kind, isRegion: Boolean, isSlack: Boolean, loc: SourceLocation)(implicit scope: Scope, flix: Flix): KindedTypeVarSym = {
+    new KindedTypeVarSym(flix.genSym.freshId(), text, kind, isRegion, isSlack, scope, loc)
   }
 
   /**
     * Returns a fresh type variable symbol with the given text.
     */
   def freshUnkindedTypeVarSym(text: VarText, isRegion: Boolean, loc: SourceLocation)(implicit scope: Scope, flix: Flix): UnkindedTypeVarSym = {
-    new UnkindedTypeVarSym(flix.genSym.freshId(), text, isRegion, scope, loc)
+    new UnkindedTypeVarSym(flix.genSym.freshId(), text, isRegion, isSlack = false, scope, loc)
```

Comment:
I had to make a choice here (and elsewhere) whether to propagate this bit to the arguments. 

But since it can only be set in very select places, I decided instead to not make it an argument, but default it to false.

its only through Type.fressEffSlackVar it is set. And that is only called in 3 places or so.

Reply:
Thoughts?

Reply:
Seems reasonable.